src/App.tsx

```diff
@@ -407,10 +417,33 @@ export default class App extends React.Component<{}, { store: any, persistor: Pe
     }
 
     public async componentDidMount() {
+        AppState.addEventListener('change', this.handleAppStateChange);
```

Comment:
It's a bit confusing that we also use our type representing store data `AppState`.